cmake/custom/ArangoDB_FindOpenSSL_WIN32.cmake

```diff
@@ -56,8 +56,11 @@ if (1909 LESS ${MSVC_VERSION} AND ${MSVC_VERSION} LESS 1920)
 elseif (1919 LESS ${MSVC_VERSION} AND ${MSVC_VERSION} LESS 1930)
 # 1920-1929: VS 16.0 (v142 toolset)
     set(VS_VERSION "2019")
+elseif (1930 LESS ${MSVC_VERSION} AND ${MSVC_VERSION} LESS 1940)
```

Comment:
```suggestion
elseif (1929 LESS ${MSVC_VERSION} AND ${MSVC_VERSION} LESS 1940)
```

Reply:
Added separetely in https://github.com/arangodb/arangodb/commit/81ccd9e624d53654e4cfc4d09dee06ba451fe7bc.